google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/resources/com/google/cloud/spanner/jdbc/it/Singers.txt

```diff
@@ -0,0 +1,30 @@
+cF92fsG1udP+6vZ+J7u9/+3ej+2cNHhTdX/bC1dH9rZeNju7PWnBudP/e/f8yun/UZ98T3b9s6r3R/XXT/y26/7MfZ7+PCwCglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKDVwyikHRh94acTj0f2jxx4f3f/QHVOi+zdcNia6/9Bh86L76w+/Jrp/0p/Pie7ftm6f6P4++x4b3b977XHR/SOvHR3dX7Thruj+C4+/Ed1ftfTV6P4d962L7rsAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSgx/65rnRB6ZufX90/80Fo6P739t+W3T/lx9fE93f8r1jo/vXzLg0uv/l3x4Q3V85+/ei+1ePXh3d/+uT/iS6P/jt46L766bMie6PfuCJ6P7ctWOi+5PXHBbddwEAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUGp37jzOgDF957V3T/5JP/Nbr/yFFD0f3fv/y86P75v1kd3Z+/18jo/uOPZH+fIzZE5991+pVfiO6fd8Z90f1XDrwjuv8/L78e3X962uLo/srJ46P7q3feE913AQCUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQY+v/a16APTnvxJdP+4H8yP7p+4ZEp0f9JvtkT35x9+TXR/yxNfiu7PPGhPdP8Dc3ZF9zePfya6P+v590T3d2zYHd1/aN7l0f0Jkz4Z3Z9087js/qQro/suAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACg1MCZtxwYfeCIz30lur9z+6To/n7LL4zuXzb9v6P7cy96Obp/64ObovsT9ro0ur98zlHR/ROWXRTdv+FvTozuTz5iQnR/1ANvRfd3feuH0f2ZH/l0dH/p2b+I7rsAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSA7PWfCb6wPS9/yO6//CKidH9BfecFN1/+uQbovtrhsZF9/c8vSa6/9Njst//9sHnovubJu4f3X/2L74R3T9l5Fej+7dtG4ruf/uT+0T37/7E9dH9R99/QXTfBQBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBrc96XvRx/YfuhN0f0nr7w+ur/1D34W3X/3zx+I7h885cXo/p3XDET3T/3UMdH9+9ZOje5fvOXy6P7dt384uj/tBwdG92fNeDu6v2veG9H9M762Iro/4bjHovsuAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACg1MCIhf8XfeAP/2s4uv/crr+L7k895J7o/oxDbo7uf3fm0uj+uaN+Et1fMfDD6P4FU7J//8b1b0f3D/3mQ9H90RecHt0f89G9o/vPf2lJdP/ji3dG9289OLvvAgAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASg186qzx0Qfe2bFfdP/B9306uv/5954T3f/iTx+O7l/yyKXR/YETn4ruH/1Pc6P7/3zyFdH9Px15RHR/8XuPje4PrZwd3T/t7KXR/f+dvCq6f9VbL0b33z3+19F9FwBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUGpw4iUfiz6wYtXY6P7s7zwf3d9xxZLo/titM6P7mz+R3Z8246Lo/jur1kf3D56+Mrp//u7d0f2Fd94f3b//6zui+xsn3xzdv3bH6dH9cXuOiO7f+p8jo/suAIBSAgBQSgAASgkAQCkBACglAAClBACglAAAlBIAgFICAFBKAABKCQBAKQEAKCUAAKUEAKCUAACUEgCAUgIAUEoAAEoJAEApAQAoJQAApQQAoJQAAJQSAIBSAgBQSgAASgkAQCkBACglAAClBACg1P8HAAD//+SLeeht2MttAAAAAElFTkSuQmCC'),DATE '1906-04-28');
```

Comment:
As discussed, let's switch the BASE64 values to something generated by `RandomResultSetGenerator`.

Reply:
The BASE64 values have been replaced with generated random numbers.